crates/goose-cli/src/prompt/cliclack.rs

```diff
@@ -184,8 +184,9 @@ impl Prompt for CliclackPrompt {
         io::stdout().flush().expect("Failed to flush stdout");
     }
 
-    fn show_busy(&self) {
-        spinner().start("awaiting reply");
+    fn show_busy(&mut self) {
+        self.spinner = spinner();
+        self.spinner.start("Thinking...");
```

Comment:
If you want to have fun with it some of the designs have had "Goose is getting a running start..."

Reply:
:) https://github.com/block/goose/pull/292